android/sdl_android/src/main/java/com/smartdevicelink/managers/lifecycle/LifecycleManager.java

```diff
@@ -155,7 +155,7 @@ void startVideoService(boolean isEncrypted, VideoStreamingParameters parameters)
      * @param isEncrypted Specify true if packets on this service have to be encrypted
      * @param parameters  VideoStreamingParameters that are desired. Does not guarantee this is what will be accepted.
      */
-    private void tryStartVideoStream(boolean isEncrypted, VideoStreamingParameters parameters) {
+    private void tryStartVideoStream(boolean isEncrypted, VideoStreamingParameters parameters, boolean afterPendingRestart) {
```

Comment:
@kostyaBoss 
Please update the documentation above to include the afterPendingRestart parameter
`@param afterPendingRestart specify true when the videoService is starting again after the display has been resized`